cubeb-core/src/ffi.rs

```diff
@@ -118,6 +118,7 @@ pub struct cubeb_stream_params {
 }
 
 #[repr(C)]
+#[derive(Clone, Copy, Debug)]
```

Comment:
Wouldn't it be safer not to implement `Copy` on any structs that contain pointers to avoid ambiguity around ownership?